server/src/main/java/com/linecorp/centraldogma/server/CentralDogma.java

```diff
@@ -398,6 +405,8 @@ private CommandExecutor startCommandExecutor(
                     return null;
                 });
             }
+            storagePurgeService.stop();
```

Comment:
I wonder if stopping the purge service will take longer time than expected if a purge operation is currently in progress. How about adding an additional log message like `Stopping the storage purge service ..`?

Reply:
Thank you, I added `stopping ...` log message before stopping.